Demos/macosAppSwiftCocoapods/macosAppSwiftCocoapods/AppDelegate.swift

```diff
@@ -22,29 +23,33 @@ class AppDelegate: NSObject, NSApplicationDelegate {
             self.callTroublemaker();
         },
         catchRun: { (exception) in
-            Rollbar.error("Got an exception!", exception: exception)
+            Rollbar.log(RollbarLevel.error, exception: exception!)
         },
         finallyRun: {
-            Rollbar.info("Post-trouble notification!");
+            Rollbar.log(RollbarLevel.info, message:"Post-trouble notification!");
         })
     }
 
     func applicationWillTerminate(_ aNotification: Notification) {
         // Insert code here to tear down your application
-        Rollbar.info("The hosting application is terminating...");
+        Rollbar.log(RollbarLevel.info, message:"The hosting application is terminating...");
     }
 
     func initRollbar() {
 
         // configure Rollbar:
-        let config = RollbarConfiguration.init();
-        
+        let config = RollbarConfig();
         //config.crashLevel = @"critical";
-        config.environment = "samples";
-        config.asRollbarConfig().customData = [ "someKey": "someValue", ];
+        config.destination.accessToken = "2ffc7997ed864dda94f63e7b7daae0f3";
```

Comment:
this looks like a real access token

Reply:
it is for the project dedicated to unit testing and demos of the Apple SDKs.

Reply:
well, usually it's not a good practice to make tokens as part of repo. you can read it from env var for demo purposes. 
for unit tests, I think you can put anything if you are testing the token setter

Reply:
besides , if this is a valid production token then somebody can use it to send many messages

Reply:
In general, agree. But here many tests hit our API to have a full functionality stack test. I am also planning to add a bit later the way Rollbar.NET does - to have a test harness send test payloads and verify a full round of outcomes via our API reply stats that should match a unit test's expectations.  But I need to refactore this SDK more before I can get there.
So far this approach was working well for the users of iOS, Apple, and .NET SDKs. It also allows them to clone an SDK repo and run/debug fully working demoes/examples right away.